src/controllers/user.controller.ts

```diff
@@ -47,20 +44,8 @@ export class UserController {
 
   @post('/users')
   async create(@requestBody() user: User): Promise<User> {
-    // Validate Email
-    if (!isemail.validate(user.email)) {
-      throw new HttpErrors.UnprocessableEntity('invalid email');
-    }
-
-    // Validate Password Length
-    if (user.password.length < 8) {
-      throw new HttpErrors.UnprocessableEntity(
-        'password must be minimum 8 characters',
-      );
-    }
-
-    // Salt + Hash Password
-    user.password = await hashAsync(user.password, 10);
+    validateCredentials(_.pick(user, ['email', 'password']));
+    user.password = await hashPassword(user.password, 4);
```

Comment:
- In production, we need to use many hash rounds to get high level of security. E.g. 10.
- In tests, we want to use as little rounds as allowed to get high speed. E.g. 4.

If it's too work to make this number customizable, then let's keep 10 for now and open a follow-up issue to address this properly.

Reply:
@bajtos Oops I shouldn't change the round number in controller. Good catch. The util function takes in the round number, flexible enough to customize it. Reverted it to 10.